### PR TITLE
Hex Cleanup

### PR DESCRIPTION
**Pull request by engineer124**
Hex with capital letters. For reference, the regex I used was `0x[0-9a-fA-F]*[a-f][0-9a-fA-F]*` incase we want to double check at a later time

### CODE REVIEW

**Review by hensldm — APPROVED** [review]

**Comment by isghj5** [comment]
Only one actor had any lower hex? that was unexpected..

**Review by isghj5 — APPROVED** [review]

File: src/boot_O2_g3/fault.c [hunk 1]
```diff
@@ -428,17 +428,17 @@ void osSyncPrintfThreadContext(OSThread* t) {
     osSyncPrintfFReg(0x16, &ctx->fp22.f.f_even);
     osSyncPrintf("\n");
     osSyncPrintfFReg(0x18, &ctx->fp24.f.f_even);
-    osSyncPrintfFReg(0x1a, &ctx->fp26.f.f_even);
+    osSyncPrintfFReg(0x1A, &ctx->fp26.f.f_even);
     osSyncPrintf("\n");
-    osSyncPrintfFReg(0x1c, &ctx->fp28.f.f_even);
-    osSyncPrintfFReg(0x1e, &ctx->fp30.f.f_even);
+    osSyncPrintfFReg(0x1C, &ctx->fp28.f.f_even);
+    osSyncPrintfFReg(0x1E, &ctx->fp30.f.f_even);
```
Comment (EllipticEllipsis) [inline src/boot_O2_g3/fault.c:434]: These should be in dec anyway.

File: src/boot_O2_g3/fault.c [hunk 2]
```diff
@@ -696,7 +696,7 @@ void Fault_ResumeThread(OSThread* t) {
     t->context.cause = 0;
     t->context.fpcsr = 0;
     t->context.pc += 4;
-    *(u32*)t->context.pc = 0xd;
+    *(u32*)t->context.pc = 0xD;
```
Comment (EllipticEllipsis) [inline src/boot_O2_g3/fault.c:699]:
```suggestion
    *(u32*)t->context.pc = 0x0000000D; // write in a break instruction
```

File: src/code/z_room.c [hunk 3]
```diff
@@ -58,7 +58,7 @@ void Room_DrawType1Mesh(PlayState* play, Room* room, u32 flags) {
     } else if (mesh->format == 2) {
         func_8012E32C(play, room, flags);
     } else {
-        __assert("../z_room.c", 0x3c5);
+        __assert("../z_room.c", 0x3C5);
```
Comment (EllipticEllipsis) [inline src/code/z_room.c:61]: This is a line number, should be dec. (all the others seem to be fine)

File: src/code/z_room.c [hunk 4]
```diff
@@ -87,7 +87,7 @@ s32 Room_StartRoomTransition(PlayState* play, RoomContext* roomCtx, s32 index) {
         size = play->roomList[index].vromEnd - play->roomList[index].vromStart;
         roomCtx->activeRoomVram =
             (void*)(((s32)roomCtx->roomMemPages[roomCtx->activeMemPage] - (size + 8) * roomCtx->activeMemPage + 8) &
-                    0xfffffff0);
+                    0xFFFFFFF0);
```
Comment (EllipticEllipsis) [inline src/code/z_room.c:90]: This is an ALIGN16

File: src/code/z_room.c [hunk 5]
```diff
@@ -114,7 +114,7 @@ s32 Room_HandleLoadCallbacks(PlayState* play, RoomContext* roomCtx) {
 
             if (((play->sceneNum != SCENE_IKANA) || (roomCtx->currRoom.num != 1)) &&
                 (play->sceneNum != SCENE_IKNINSIDE)) {
-                play->envCtx.lightSettingOverride = 0xff;
+                play->envCtx.lightSettingOverride = 0xFF;
```
Comment (EllipticEllipsis) [inline src/code/z_room.c:117]: This ought to get a define, OoT has `LIGHT_SETTING_OVERRIDE_NONE` in `z64environment.h`.

**Review by EllipticEllipsis — COMMENTED** [review]
The problem with doing PRs like this is they tend to unearth places where we still have a lot of work to do. I'm also not sure to what extent it's worth touching Libultra with this sort of thing.

File: src/code/z_room.c (hunk 5 above)
Comment (engineer124) [reply src/code/z_room.c:117]: I want to keep this PR focused to just hex numbers, so we'll add it to the to-do list

File: src/code/z_room.c (hunk 4 above)
Comment (engineer124) [reply src/code/z_room.c:90]: I think it's missing the `+ 0xF` to be an `ALIGN16`, but I can change it to `~0xF`

Comment (EllipticEllipsis) [reply src/code/z_room.c:90]: OoT has
```c
roomCtx->unk_34 = (void*)ALIGN16((u32)roomCtx->bufPtrs[roomCtx->unk_30] - ((size + 8) * roomCtx->unk_30 + 7));
```

Comment (engineer124) [reply src/code/z_room.c:90]: Ah I see. Has to be `- 7` but same concept. Yeah it works

Comment (EllipticEllipsis) [reply src/code/z_room.c:90]: Strange that it's different at all, will be interesting to compare the retail asm.

**Review by EllipticEllipsis — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**